automated install/basic-install.sh

```diff
@@ -2114,7 +2114,7 @@ main() {
   fi
 
   # Display where the log file is
-  echo -e "\\n  ${INFO} The install log is located at: /etc/pihole/install.log
+  echo -e "\\n  ${INFO} The install log is located at: /var/log/pihole-install.log
```

Comment:
@Mcat12 I suppose we could change the filename to `pihole_install.log`, too.  But it should also be used as a variable here.  I'm sure there are lots of cases like this (hardcoding something that is already a variable).  It's a minor change, but probably still worthwhile to do as a best practice.

Reply:
@jacobsalmela : Changed the static text to variable.

Reply:
You also want to change the minus sign to an underscore? Or keep that change for later?

Reply:
Might as well make the change now :)